src/Comments/Comments.js

```diff
@@ -55,15 +56,24 @@ export class Comments extends Component  {
               />
 
               <button 
-              role="commnt-button"
-              type="commnt-button"
+              role="comment-button"
+              type="comment-button"
               onSubmit={this.postComment}
               onClick={this.postComment}
               
               className="comment-button">
               add comment
               </button>
 
+              <button 
+              role="clear-button"
+              type="clear-button"
+              onClick={this.clearForm}
+              name="comment"
+              className="clear-button">
+              clear
+              </button>
```

Comment:
Good thinking adding this! It definitely adds to the UI

Reply:
nice!